mautrix_facebook/portal.py

```diff
@@ -1445,6 +1447,22 @@ async def _convert_mqtt_attachment(
         message_text: str,
     ) -> MessageEventContent:
         filename = attachment.file_name
+
+        if filename is None:
+            msg = html = "Unsupported attachment"
+            msgtype = MessageType.NOTICE
+            if attachment.mime_type == "p2p_payment":
+                sa = attachment.parse_extensible().story_attachment
+                msg = sa.title + "\n" + sa.subtitle
+                subtitle = escape(sa.subtitle.replace("\n", "<br />"))
+                html = f"<b>{escape(sa.title)}</b><br>{subtitle}"
+                msgtype = MessageType.TEXT
+            else:
+                self.log.warning(msg)
```

Comment:
That log seems fairly useless

```suggestion
                self.log.warning(f"Unsupported attachment in {msg_id} (mime: {attachment.mime_type})")
```